components/service/sync-logins/src/main/java/mozilla/components/service/sync/logins/SyncableLoginsStorage.kt

```diff
@@ -20,20 +20,19 @@ import mozilla.components.support.sync.telemetry.SyncTelemetry
 import mozilla.components.support.utils.logElapsedTime
 import org.json.JSONObject
 import java.io.Closeable
-import mozilla.appservices.logins.LoginsStorage as RustLoginStorage
 
 const val DB_NAME = "logins.sqlite"
 
 /**
  * This type contains the set of information required to successfully
  * connect to the server and sync.
  */
-typealias SyncUnlockInfo = mozilla.appservices.logins.SyncUnlockInfo
+//typealias SyncUnlockInfo = mozilla.appservices.logins.SyncUnlockInfo
```

Comment:
We need to make a decision syncing. Given you've completely removed the `sync` function and everything builds we should kill all references to SyncUnlockInfo too. Note however there are a few other comments in a-s which refer to our SyncUnlockInfo which should probably be updated (even if just to say "the (now removed) logins SyncUnlockInfo" or similar (a grep will make that a bit clearer).

cc @grigoryk - nuking stand-alone sync support from `SyncableLoginsStorage` sounds ok?

Reply:
I tried to keep it inline with autofill to keep things as close as possible so I might've went a little aggressive on removing that (and seeing nothing breaks) :). I'll make sure to update the a-s side of things too where this shows up though